### ISSUE TITLE
split() is deprecated in PHP 5.3

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/minify#119]
```
Minify version: 2.1.2
PHP version: 5.3RC4

What steps will reproduce the problem?
1. Run Minify
2. Get warning about deprecated split() in ConditionalGet.php line 292

Expected output: no warnings


Starting with PHP 5.3 (currently in RC4) the regex module is deprecated. 
The split() function is used once in ConditionalGet.php line 292, and can
be replaced with explode() or preg_split().  (explode is faster, since
there's no regex used here.)

There may be other PHP 5.3 issues, but I haven't found them yet.  See
http://cvs.php.net/viewvc.cgi/php-src/UPGRADING?revision=PHP_5_3 for a list
of changes.
```

---

Original issue reported on code.google.com by `p...@mdsc.com` on 22 Jun 2009 at 3:54

**Comment by GoogleCodeExporter** [comment]
```
Will change to explode in next release. Thanks for the catch.
```

---

Original comment by `mrclay....@gmail.com` on 22 Jun 2009 at 4:03
- Changed state: **Accepted**
- Added labels: **Release-2.1.2**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in R333/2.1.3
```

---

Original comment by `mrclay....@gmail.com` on 1 Jul 2009 at 7:04
- Changed state: **Verified**
